### ISSUE TITLE
Google Analytics - Debug Subnetclass

### ISSUE THREAD

**Issue by johnjung** [uchicago-library/library_website#31]
A while back Google Analytics switched from using custom variables to custom dimensions and metrics. We never updated our tracking codes to reflect this change. Additionally, even when we have Google Analytics set up to record this field, it doesn't always get recorded. 

100% of pageviews should get this dimension. Look at why it isn't being recorded everywhere, starting from www.lib where I have been updating the tracking code. 

Once I'm confident that this is working correctly put the fix in all of our GA scripts.

**Comment by johnjung** [comment]
If I add "sessions=1" to my custom segment in GA the number of homepage views without a subnetclass attached drops pretty dramatically--it's much less than 1%. Maybe there is a caching issue  where people have old versions of ga.public.js? Pay attention to this over the next few days and see if this number drops even further.

**Comment by johnjung** [comment]
The percentage of sessions where a subnetclass isn't being recorded keeps dropping, so I'm considering this problem solved. I checked a few of our other sites, like century.lib, chicagoan.lib, klutznick.lib, news.lib and the catalog, and I see that they have correctly implemented tracking for the subnetclass. 

I added tracking to the Campus Publications, goodspeed.lib, photoarchive.lib, and roseandchess.lib. 

I skipped some of our tracking codes- it doesn't seem especially useful to track this information for Loop, for one. speculum.lib uses prototype.js instead of jquery, so implementing it would have taken a little extra time. Some other sites, like DSAL, aren't maintained by us and would require coordination with their maintainers to make a change like this.
